tests/unit/lms/views/helpers/_via_test.py

```diff
@@ -44,16 +43,55 @@ def test_if_creates_the_correct_via_url(self, pyramid_request):
             ),
         ),
     )
-    def test_it_merges_params_correctly(self, pyramid_request, params, expected_extras):
+
+    @pytest.mark.usefixtures("legacy_via_feature_flag")
+    @pywb_test_params
+    def test_it_merges_params_correctly_for_legacy_via(
+        self, pyramid_request, params, expected_extras
+    ):
         final_url = via_url(
             pyramid_request, f"http://doc.example.com/?{urlencode(params, doseq=True)}"
         )
 
+        assert final_url == Any.url.with_host("test_legacy_via_server.is")
+        assert final_url == Any.url.with_path("/http://doc.example.com/")
```

Comment:
I expanded these tests to check we were going to the right host and path. It was a bit wishy washy before